### ISSUE TITLE
Terror from the Deep makes Worldbuilding Knowledge Base illegible

### ISSUE THREAD

**Issue by MGatta** [worldanvil/worldanvil-bug-tracker#702]
**World Anvil Username**: MGatta

**Feature**: Worldbuilding Knowledge Base

**Describe the Issue**
The text is not legible on Terror from the Deep due to lack of colour contrast.

**To Reproduce**
1. Go to Interface settings
2. Select Terror from the Deep
3. Go to https://www.worldanvil.com/knowledge-base
4. See colour issue leading to legibility issue

**Expected behavior**
An improved colour scheme that does not make the text difficult to read.

**Screenshots**
![image](https://user-images.githubusercontent.com/80196715/142753683-bf6e4ef2-f3b4-481d-971a-252688f28e8f.png)

**Comment by iamromeo** [comment]
Painfully and totally aware, this whole system is build from the ground up again. It was never really used by anyone sadly as it stands
